validator/rules_src/uuid1.py

```diff
@@ -0,0 +1,36 @@
+from validator.rules_src import Rule
+from uuid import UUID
+
+
+class UUID1(Rule):
+    """
+    The field under validation must be formatted as an uuid1
+
+    Examples:
+    >>> from validator import validate
+
+    >>> reqs = {'data' : 'eb241bb4-c087-11ea-b3de-0242ac130004'}
+    >>> rule = {'data' : 'uuid1'}
+    >>> validate(reqs, rule)
+    True
+    >>> reqs = {'data' : 'bba617b4-364b-4a0d-9e96-cb8a24ef1bec'}
+    >>> rule = {'data' : 'uuid1'}
+    >>> validate(reqs, rule) # False, it's uuidv4
+    False
+    """
+
+    def __init__(self):
+        Rule.__init__(self)
+
+    def check(self, arg):
+        try:
+            uuid = UUID(arg, version=1)
+            return str(uuid) == arg
```

Comment:
Please remove `version=1` argument, just create with `uuid = UUID(arg)` afterwards check if `uuid.version` is equal to 1. if not equal please set error message.

Reply:
also note that `.version` returns None if incorrect UUID is passed